### DISCUSSION TITLE
How to apply to be a member of the ESLint team?

### DISCUSSION THREAD

**Discussion by JounQin** [eslint/eslint#16107] [General] [answered]
As title, I'd like to apply to be a member of the ESLint team, is there any chance?

I'm working on a lot of ESLint related packages like [`eslint-mdx`](https://github.com/mdx-js/eslint-mdx), [`eslint-import-resolver-typescript`](https://github.com/import-js/eslint-import-resolver-typescript), [`eslint-plugin-prettier`](https://github.com/prettier/eslint-plugin-prettier), etc.

I'd like to help to improve the core package in the further with the team together.

**Comment by harish-sethuraman** [comment, marked as answer]
You can find the process here https://eslint.org/docs/latest/maintainer-guide/governance#committers

**Reply by JounQin** [reply]
Thanks for pointing out!
